src/upf_network.py

```diff
@@ -212,8 +283,13 @@ def configure(self) -> None:
 
     def is_configured(self) -> bool:
         """Return whether the network is configured for the UPF service."""
-        return (
+        ifaces_are_configured = (
```

Comment:
This has me thinking: we configured the interfaces, but on removal, we leave the configuration behind. It is too much to expect us to return the iface to its prior state, but we should have an `_on_remove` type handler to delete the routes and interfaces that we set.

Reply:
I could add an `unset_ip_address` to `NetworkInterface` class to remove the configured IP.
The same could be done for the routes I _think_.
Let me know so I can address those before merging.

Reply:
Let's do that now. The integration tests are failing due to the `up4.bess` script hanging.